changelog.yaml

```diff
@@ -52,6 +52,15 @@
   date: TBD
   changes:
 
+    - type: enhancement
+      impact: minor
+      title: Make PodSecurityPolicies for pipelineRun and both controllers configurable
+      description: |-
+        To support Kyma pipelines 'PodSecurityPolicy' should be configurable so that from the installer or backend system can be changed.
```

Comment:
```suggestion
        To support Kyma pipelines 'PodSecurityPolicy' should be configurable so that it can be changed during installation with helm.
```